integration/spark/shared/src/main/java/io/openlineage/spark/agent/facets/builder/DatabricksEnvironmentFacetBuilder.java

```diff
@@ -110,7 +110,7 @@ private Map<String, Object> getDatabricksEnvironmentalAttributes(SparkListenerJo
   private static Optional<DbfsUtils> getDbfsUtils()
       throws ClassNotFoundException, InstantiationException, IllegalAccessException,
           IllegalArgumentException, InvocationTargetException {
-    Class dbutilsClass = Class.forName("com.databricks.dbutils_v1.impl.DbfsUtilsImpl");
+    Class dbutilsClass = Class.forName("com.databricks.sdk.scala.dbutils.SdkDBUtilsImpl");
```

Comment:
What it looks to me, we should remove `throws ClassNotFoundException` from method signature, catch the exception and return `Optional.empty` if so. Moreover, it may have sense to support both class names and choose the one that is present.

Reply:
It's good idea to remove the `throws ClassNotFoundException` but we should log some warning that the Databricks Facet is not built because the package is not present in the classpath.
For me, I think we can keep `com.databricks.dbutils_v1.impl.DbfsUtilsImpl` which already exists by default and simply remove `com.databricks.sdk.scala.dbutils.SdkDBUtilsImpl`.

Tell me that do you think.

Reply:
I have one question, would the warning be logged in non-Databricks environments? If the answer is yes, then it doesn't make sense to log the warning.

Reply:
Something similar ti what we see [here](https://github.com/OpenLineage/OpenLineage/blob/c82c208f886ab57e0299f904d665bac74b413856/integration/spark/app/src/main/java/io/openlineage/spark/agent/lifecycle/SparkSQLExecutionContext.java#L81) It goes in the Log4j output alongside with spark's logs.